src/code-helpers/style-object-helpers.js

```diff
@@ -38,7 +47,14 @@ function generateShadowStyleObject({
 
     // "iOS" doesn't have shadow spread
     return {
-        shadowColor: getColorStringByFormat(shadow.color, colorFormat),
+        shadowColor: getColorValue(
+            shadow.color,
+            {
+                container,
+                useLinkedStyleguides,
+                colorFormat
+            }
+        ),
```

Comment:
We need to pass these parameters everywhere a color value is needed (or `getColorValue` used). 

Instead of moving a set of parameters around, we can create a context-aware container (project/styleguide) object so that it should have everything it needs to know about the context (such as options, etc.). This way we can use this specialized container to get a color value based on the context just by providing the color object.

For example:
```suggestion
        // container object should have been initialized already and it knows target color format, styleguide access, etc.
        shadowColor: container.getColorValue(shadow.color),
```

Reply:
I wrapped context object. You can check the latest version. WDYT @dirtybit?